src/main/java/com/spotinst/sdkjava/model/bl/ocean/kubernetes/ClusterStrategyConfiguration.java

```diff
@@ -12,6 +12,7 @@
     private Boolean     utilizeReservedInstances;
     private Integer     drainingTimeout;
     private Integer     gracePeriod;
+    private Integer     spotPercentage;
```

Comment:
Don't use the same Cluster level class. It has different parameters. Create another class for Vng level.